core/src/components/modal/modal-interface.ts

```diff
@@ -33,4 +33,12 @@ export interface ModalAnimationOptions {
   backdropBreakpoint?: number;
 }
 
-export interface ModalAttributes extends JSXBase.HTMLAttributes<HTMLElement> {}
+export interface ModalAttributes extends JSXBase.HTMLAttributes<HTMLElement> { }
+
+export interface ModalBreakpointChangeEventDetail {
+  breakpoint: number;
+}
+
+export interface ModalCustomEvent extends CustomEvent {
```

Comment:
Ok sorry last comment: Can we also document this in the readme? See the infinite scroll readme for an example of how we document it.